src/reducers/notesReducerFunction.js

```diff
@@ -3,15 +3,17 @@ const actionTypes = {
 	SET_NOTES_ERROR: "SET_NOTES_ERROR",
 	RESET_NOTES: "RESET_NOTES",
 	SHOW_NEW_NOTE_FORM: "SHOW_NEW_NOTE_FORM",
-	EDIT_NOTE: "EDIT_NOTE",
+	SET_ARCHIVES: "SET_ARCHIVES",
+	EDIT_ARCHIVES: "EDIT_ARCHIVES",
```

Comment:
Usually these constants are stored in a separate file named actions.js

Reply:
Yes, I have moved them.